packages/theme/src/colors.ts

```diff
@@ -9,6 +9,7 @@ const defaultColorVariant = {
     yellow: '#EFC941',
     blue: '#0078AC',
     red: '#CD4949',
+    red5: 'rgba(205, 73, 73, 0.05)',
```

Comment:
this color is not used anywhere? 

also always consult adding theme color with Ondra graphics guy, because we need to keep our palette 1:1 with Figma and he will have to add color, and figure colors for othere theme variants etc. 

one more point, I think we have transparentize function in utils `utils.transparentize(0.05)(utils.colors.red)` should do same job

Reply:
[0585e93](https://github.com/trezor/trezor-suite/pull/5433/commits/0585e93bf57a1592805b2c03261c0fd25665537c)

I used it when I thought I'm doing the form input issues together with this one. I removed it for now and will solve it better in next PR.